Do not use -s flag. clang says it ignores it, yet with some versions of
clang the build then fails.
--- setup.py.orig	2009-03-14 12:32:21.000000000 -0500
+++ setup.py	2020-01-12 12:06:26.000000000 -0600
@@ -16,8 +16,6 @@
 ext_modules=[ 
     Extension('clnum.clnum', ['clnum/src/clnum.cpp', ],
         libraries=['cln', 'gmp'],
-        # Remove symbols.
-        extra_link_args=['-s'],
     ),
 ],
 )
